resources/js/Molecules/FormRadioButton.tsx

```diff
@@ -23,20 +25,20 @@ const FormRadioButton: React.FC<IProps> = ({
 }) => {
 	return (
 		<div>
-			<label className="block text-sm leading-5 font-medium text-gray-700">{label}:</label>
+			<label className="block text-sm leading-5 font-medium text-gray-700">{label}</label>
 			<p className="text-sm leading-5 font-medium text-gray-400">
 				{helpText}
 			</p>
 			{choices &&
 				choices.map((choice, index) => (
 					<div key={index} className="mt-4 flex items-center">
 						<input
-							type="radio"
+							type={type}
 							id={choice.value}
 							name={name}
 							value={choice.value}
-							checked={selected == choice.value}
-							className="form-radio h-4 w-4 text-indigo-600 transition duration-150 ease-in-out"
+							checked={type==="radio"?selected == choice.value:selected.indexOf(choice.value) > -1}
```

Comment:
when you check for equality, it is good practice to use the triple equal sign
selected ===  choice.value